apps/safe-claiming-app/src/hooks/useSafeTokenAllocation.ts

```diff
@@ -141,11 +141,21 @@ const fetchTokenBalance = async (
   }
 }
 
+const computeVotingPower = (allocationData: Vesting[], balance: string) => {
+  const tokensInVesting = allocationData.reduce(
+    (acc, data) =>
+      data.isExpired ? acc : acc.add(data.amount).sub(data.amountClaimed),
+    BigNumber.from(0)
+  )
+
+  // add balance
+  return tokensInVesting.add(balance || "0")
+}
```

Comment:
Can you add a quick test for this?

Reply:
This is tested through the `useSafeTokenAllocation.test.ts`

I just refactored it out into its own function here to have it better readable.